ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/CVEsTable.tsx

```diff
@@ -150,16 +149,13 @@ function CVEsTable({
                             isFixable,
                             cveType,
                             cvss,
-                            scoreVersion,
-                            distroTuples,
+                            clusterVulnerability: { summary, scoreVersion },
```

Comment:
Platform CVEs do not have multiple associated distros, so this query uses the singular `clusterVulnerability` resolver instead of the `distroTuples` array resolver.